etlman/projects/tests/test_views/test_view_dropdown.py

```diff
@@ -0,0 +1,65 @@
+from http import HTTPStatus
+from random import randint
+
+import pytest
+from django.urls import reverse
+
+from etlman.projects.tests.factories import ProjectFactory
+
+
+@pytest.mark.django_db
+class TestProjectDropdownList:
+    def test_dropdown_empty(self, nonadmin_client):
+        response = nonadmin_client.get(reverse("home"))
+        html = str(response.content)
+        assert response.status_code == HTTPStatus.OK.numerator
+        assert "Add a project" in html, html
+
+    def test_dropdown_with_one_project(self, nonadmin_client):
+        project_data = ProjectFactory.build()
+        data = {"name": project_data.name, "description": project_data.description}
+        response = nonadmin_client.post(
+            reverse("projects:new_project"),
+            data=data,
+        )
+        response = nonadmin_client.get(reverse("home"))
+        html = str(response.content)
+        assert response.status_code == HTTPStatus.OK.numerator
+        assert "Add a project" in html, html
+        assert project_data.name in html
+
+    def test_dropdown_with_many_projects(self, nonadmin_client):
+        project_list = ProjectFactory.build_batch(randint(1, 10))
+        for project_data in project_list:
+            data = {"name": project_data.name, "description": project_data.description}
+            response = nonadmin_client.post(
+                reverse("projects:new_project"),
+                data=data,
+            )
+        response = nonadmin_client.get(reverse("home"))
+        html = str(response.content)
+        assert response.status_code == HTTPStatus.OK.numerator
+        assert "Add a project" in html, html
+        assert all([project_data.name in html for project_data in project_list])
+
+    # def test_dropdown_with_many_projects_without_other_user_projects(self, nonadmin_client):
+    #     client1 = get_authenticated_client("testuser")
+    #     project_list = ProjectFactory.build_batch(randint(1, 10))
```

Comment:
Since this is just testing the visibility, no need to use the view to create these. Just use the factory instead.
```suggestion
    #     project_list = ProjectFactory.create_batch(randint(3, 6))
```